RNGNewAuraNotifier/Core/Json/JsonData.cs

```diff
@@ -45,8 +46,9 @@ public static async Task GetLatestJsonDataAsync()
     /// <returns>JSONファイルの内容</returns>
     public static JsonData GetJsonData()
     {
+        ConfigData configData = AppConfig.Instance;
         // Jsonファイルの保存先
-        var jsonDir = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), "RNGNewAuraNotifier", "Aura.json");
+        var jsonDir = Path.Combine(configData.AuraJsonDir, "Aura.json");
```

Comment:
ディレクトリに対するパスではないので、Dirではない方がいいかもです。

```suggestion
        var jsonFilePath = Path.Combine(configData.AuraJsonDir, "Aura.json");
```